DataFormats/TPC/include/DataFormatsTPC/ClusterNative.h

```diff
@@ -43,24 +45,28 @@ struct ClusterNative {
   void setTimeFlags(float time, uint8_t flags) {mTimeFlagsPacked = (((uint32_t) (time * mScaleTimePacked + 0.5)) & 0xFFFFFF) | ((uint32_t) flags << 24);}
   float getPad() const {return mPadPacked / mScalePadPacked;}
   void setPad(float pad) {mPadPacked = (uint16_t) (pad * mScalePadPacked + 0.5);}
-  float getSigmaTime2() const {return mSigmaTime2Packed / mScaleSigmaTimePacked;}
-  void setSigmaTime2(float sigmaTime2) {uint32_t tmp = sigmaTime2 * mScaleSigmaTimePacked + 0.5; if (tmp > 0xFF) tmp = 0xFF; mSigmaTime2Packed = tmp;}
-  float getSigmaPad2() const {return mSigmaPad2Packed / mScaleSigmaPadPacked;}
-  void setSigmaPad2(float sigmaPad2) {uint32_t tmp = sigmaPad2 * mScaleSigmaPadPacked + 0.5; if (tmp > 0xFF) tmp = 0xFF; mSigmaPad2Packed = tmp;}
+  float getSigmaTime() const {return mSigmaTimePacked / mScaleSigmaTimePacked;}
+  void setSigmaTime(float sigmaTime) {uint32_t tmp = sigmaTime * mScaleSigmaTimePacked + 0.5; if (tmp > 0xFF) tmp = 0xFF; mSigmaTimePacked = tmp;}
```

Comment:
this has been here before this PR, I'm wondering why the setter has a 0.5 shift. It's probably irrelevant since the scale factor is an integral value even though not of integral type. Maybe all the factors could be of integral types.

Reply:
but the sigma itself is a float, so we need the 0.5 for correct rounting.